src/RelayServer.ts

```diff
@@ -259,15 +258,15 @@ export class RelayServer extends EventEmitter {
     }
   }
 
-  async validateInput(envelopingRequest: EnvelopingTxRequest): Promise<void> {
+  async validateInput(envelopingRequest: HttpEnvelopingRequest): Promise<void> {
     const { metadata, relayRequest } = envelopingRequest;
 
     const {
       app: { requestMinValidSeconds },
       contracts: { relayHubAddress },
     } = this.config;
 
-    const relayHubAddressValue = await metadata.relayHubAddress;
+    const relayHubAddressValue = metadata.relayHubAddress;
```

Comment:
We could also, instead of accessing the value, destruct it. In the previous section.

Reply:
Done